tools/mo/openvino/tools/mo/convert.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from collections import namedtuple
+from openvino.tools.mo.convert_impl import _convert
+
+InputCutInfo = namedtuple("InputInfo", ["name", "shape", "type", "value"])
+LayoutMap = namedtuple("LayoutMap", ["source_layout", "target_layout"])
+
+
+def convert(**args):
```

Comment:
Add one positional argument for models which consists of one file only. It will allow us omitting boring `input_model`.

Reply:
Done. Please check.